### PR TITLE
Add branch priority

### PR DESCRIPTION
**Pull request by jmTUD**
This pull request adds a pure wrapper of the SCIP function "SCIPchgVarBranchPriority" from "scip/scip_var.h".
The python version can be accessed from `Model`.chgVarBranchPriority(`Variable` var, priority).

No tests/examples are provided, since it is a direct wrapper of a SCIP method.

### CODE REVIEW

**Comment by fserra** [comment]
Hi thanks for the contribution, could you merge master into your branch so that the test can pass.

File: src/pyscipopt/scip.pyx [hunk 1]
```diff
@@ -4537,6 +4537,15 @@ cdef class Model:
 
         free(_coeffs)
 
+    def chgVarBranchPriority(self, Variable var, priority):
+        """Set the branch priority of a variable.
```
Comment (fserra) [inline src/pyscipopt/scip.pyx:4541]: can you copy verbose the documentation from scip, please:

> sets the branch priority of the variable; variables with higher branch priority are always preferred to variables with lower priority in selection of branching variable

Comment (jmTUD) [reply src/pyscipopt/scip.pyx]: Thank you for your comment. I changed it.

**Comment by fserra** [comment]
thanks for your contribution!

**Pull request merged**